packages/vaadin-charts/theme/vaadin-chart-default-theme.js

```diff
@@ -203,6 +206,12 @@ registerStyles(
       stroke-linejoin: round;
     }
 
+    .highcharts-empty-series {
```

Comment:
Are there visual tests for these styles? Do we need to create some?

Reply:
The default theme is `highcharts.css` with few additions to include the `--vaadin-charts-color-#` variables. It's only updated when bumping highcharts version. We don't have visual tests covering all possible chart types and configurations.